docs/parser.md

```diff
@@ -3,4 +3,22 @@ title: "Parser"
 weight: 90
 ---
 
->This document is under construction.
\ No newline at end of file
+## Parser
+
+AsyncAPI Parser is a package used to parse and validate the AsyncAPI documents in your Node.js or browser application. These documents can be either in–YAML or JSON format.
+
+Parser validates these documents using dedicated schema-supported plugins. These supported schemas are:
+
+- AsyncAPI schema
+- OpenAPI schema
+- JSON schema
+- Avro schema
+- RAML data-type schema(schema parser for RAML data types)
+
+> Note: If you use the parser as a package, you can also register external schemas. For example, you can also write your own schema.
```

Comment:
It would be great to get a better example here.

Reply:
What would be a better example?

Reply:
An actual code example of how you use these plugins with the parser. See https://github.com/asyncapi/avro-schema-parser for an example.

Reply:
At the very least I'd link to an example on the Parser docs. Otherwise, this comment here seems unhelpful to me.

Reply:
hey @fmvilas @derberg I have added a demo json example which modifies the streetlight one. Though im not really sure if its correct🤔 Can you check it out